tests/common/helpers/voq_lag.py

```diff
@@ -60,34 +60,28 @@ def verify_lag_interface(duthost, asic, portchannel, expected=True):
     return False
 
 
-def add_lag(duthost, asic, portchannel_members=None, portchannel_ip=None,
-            portchannel=TMP_PC, add=True):
-    """
-    Add LAG to an ASIC
-    runs command e.g. 'sudo config portchannel -n asic0 add PortChannel99'
-    Args:
-        duthost<object>: duthost
-        asic<object>: asic object
-        portchannel_members<list> : portchannel members
-        portchannel_ip<str>: portchannel ip
-        portchannel<str> : portchannel
-        add<bool> : True adds portchannel
-    """
-    if add:
-        config_cmd = "config portchannel {}"\
-            .format(asic.cli_ns_option if asic.cli_ns_option else "")
-        duthost.shell("{} add {}".format(config_cmd, portchannel))
-        int_facts = duthost.interface_facts(namespace=asic.namespace)['ansible_facts']
-        pytest_assert(int_facts['ansible_interface_facts'][portchannel])
+def add_lag(duthost, asic, portchannel=TMP_PC):
+    """Creates a LAG on given ASIC"""
+    logging.info("Adding LAG {} to {} asic{}".format(portchannel, duthost, asic.asic_index))
+    duthost.shell("config portchannel {} add {}".format(asic.cli_ns_option, portchannel))
 
+
+def delete_lag(duthost, asic, portchannel=TMP_PC):
+    """Deletes a LAG on given ASIC"""
+    logging.info("Deleting lag from {}".format(duthost.hostname))
+    duthost.shell("config portchannel {} del {}".format(asic.cli_ns_option, portchannel))
+
+
+def add_members_ip_to_lag(duthost, asic, portchannel_members=None, portchannel_ip=None, portchannel=TMP_PC):
+    """Add members and IP to LAG"""
     if portchannel_members:
+        logging.info("Adding members {} to LAG {}".format(portchannel_members, portchannel))
         for member in portchannel_members:
-            duthost.shell("config portchannel {} member add {} {}"
-                          .format(asic.cli_ns_option, portchannel, member))
+            duthost.shell("config portchannel {} member add {} {}".format(asic.cli_ns_option, portchannel, member))
 
     if portchannel_ip:
-        duthost.shell("config interface {} ip add {} {}"
-                      .format(asic.cli_ns_option, portchannel, portchannel_ip))
+        logging.info("Assigning IP {} to LAG {}".format(portchannel_ip, portchannel))
+        duthost.shell("config interface {} ip add {} {}".format(asic.cli_ns_option, portchannel, portchannel_ip))
```

Comment:
use the function in sonic_asic.py https://github.com/sonic-net/sonic-mgmt/blob/9b673bb1f17d201c9698a43720df47aa95d4b0d6/tests/common/devices/sonic_asic.py#L517